bundles/org.eclipse.equinox.launcher/src/org/eclipse/equinox/launcher/Main.java

```diff
@@ -560,6 +560,13 @@ private void basicRun(String[] args) throws Exception {
 		// the location of the boot plugin we are going to use
 		handleSplash();
 
+		try {
```

Comment:
I think one would at least want a system property to disable this behavior without the need to configure an own keystore.